UnitsNet.Tests/CustomCode/FlowTests.cs

```diff
@@ -48,5 +48,7 @@ public class FlowTests : FlowTestsBase
         protected override double DecilitersPerMinuteInOneCubicMeterPerSecond => 600000.00000;
 
         protected override double KilolitersPerMinuteInOneCubicMeterPerSecond => 60.00000;
+
+        protected override double MillionStandardCubicFeetPerDayInOneCubicMeterPerSecond => 3.058175869;
```

Comment:
According to [this converter](http://www.traditionaloven.com/tutorials/flow-rate/convert-mil-standard-ft3-day-to-m3-cubic-meter-per-second.html) I find this should be `0.3277412847222222`. Double checked here http://www.onlineconversion.com/forum/forum_1120558694.htm.